lorien/InfiniteCanvas/Tools/CanvasTool.gd

```diff
@@ -13,6 +13,9 @@ var _cursor: BaseCursor
 var _canvas: InfiniteCanvas
 var enabled := false: get = get_enabled, set = set_enabled
 var performing_stroke := false
+var disable_stroke := false
```

Comment:
Tools can be enabled/disabled with `set_enabled(bool)`, but this introduces another way to disable it. I assume it's so that you can still see the cursor. This is fine, but i wonder if this variable could be named differently to reflect that and not be confused with the set_enabled method.

Reply:
Yes, to still be able to see the cursor is indeed nice to have. Moreover, if I use `set_enabled(bool)` instead (still in `CanvasTool.gd`), this will put a crosshair into the cursor which can't be get rid off except by restarting Lorien or switching to the selection tool and back again.
![image](https://github.com/user-attachments/assets/03f45e7f-b03c-4049-b15b-a72a5f6f05d7)

I find `disable_stroke` fitting. Since the tool is in principle still enabled that way, I personally don't find it confusing. Another idea for name I have would be `disable_drawing`, but I don't like that one as much. 🤔

Reply:
After thinking about it for a while, i agree. `disable_stroke` is good.